src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ReleaseArrangementLookupServiceTest.kt

```diff
@@ -332,7 +333,7 @@ class ReleaseArrangementLookupServiceTest {
 
     val unsupportedSentence = nonSDSPlusSentenceAndOffenceFourYears.copy(
       sentenceCalculationType = SentenceCalculationType.entries.find {
-        !it.isSDS40Eligible
+        it.sentenceType.sentenceClazz != StandardDeterminateSentence::class.java
```

Comment:
Can these two changes in this file be reverted?

Reply:
No, but I've reworked it to use the SentenceCalculationType functions